UtopiaCity/Extensions/ServiceAdder.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using UtopiaCity.Services.FireDepartment.FireIncidentReportServices;
+
+namespace UtopiaCity.Extensions
+{
+    public static class ServiceAdder
```

Comment:
when this pattern is used it is usually used like this - 1 class - 1 logical area - basically, one extension class per 1 person in our case, lets discuss this on wednesday